src/assets/styles/fonts/_icons.scss

```diff
@@ -30,7 +30,7 @@ $icomoon-font-family: 'icons';
   }
 }
 
-// FontAwesomness
+// FontAwesomeness
```

Comment:
Sorry, must have missed this one — that's [dashboard](https://github.com/rancher/dashboard/blob/2345527a916cde3909a5ca0ceae3b7b36f593603/assets/styles/fonts/_icons.scss#L31) too.